src/snowcli/utils.py

```diff
@@ -570,7 +570,8 @@ def convert_resource_details_to_dict(function_details: SnowflakeCursor) -> dict:
 
 def generate_deploy_stage_name(identifier: str) -> str:
     return (
-        identifier.replace(
+        identifier.replace("()", "")
```

Comment:
1. Why this change is required?
2. Do I see correctly that this change will affect also functions and procedures, not only procedure coverages? If yes, then the release notes should be updated.

Reply:
1. In previous PR, I changed replace `(` to `_`, but later I found that there is second function doing almost the same thing (one of them is lowering result). When I changed replace in second function, tests with no parameters began to fail.